app/src/app_config.py

```diff
@@ -19,3 +24,10 @@ class AppConfig(PydanticBaseEnvConfig):
     host: str = "127.0.0.1"
     port: int = 8080
     chat_engine: str = "guru-snap"
+
+    def db_session(self) -> db.Session:
+        return db.PostgresDBClient().get_session()
+
+    @cached_property
+    def sentence_transformer(self) -> SentenceTransformer:
+        return SentenceTransformer(self.embedding_model)
```

Comment:
`AppConfig` functions as a context object that provides access to the DB and `sentence_transformer` embedding model instance.